plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -12,55 +12,25 @@ use alumet::{
 
 pub(crate) const CGROUP_MAX_TIME_COUNTER: u64 = u64::MAX;
 
-/// # Structure
-///
-/// * `CgroupV2Metric` - Public structure storing CPU and memory data
-///
-/// # Parameters
-///
-/// * `name` - Name of a Kubernetes pod
-/// * `uid` - Unique identification of a Kubernetes pod
-/// * `namespace` - Resources isolation of a Kubernetes pod
-/// * `node` - Kubernetes pod node
-/// * `time_used_tot` - Total CPU usage time by the cgroup
-/// * `time_used_user_mode` - CPU in user mode usage time by the cgroup
-/// * `time_used_system_mode` - CPU in system mode usage time by the cgroup
-/// * `anon_used_mem` - Anonymous used memory, corresponding to running process and various allocated memory
-/// * `file_mem` - Files memory, corresponding to open files and descriptors
-/// * `shared_mem` - Interprocess communication shared memory
-/// * `file_mapped_mem` - Mapped files in memory
-/// * `total_mem` - Total memory used by cgroup
-///
 #[derive(Debug, PartialEq, Clone)]
 pub struct CgroupV2Metric {
-    pub name: String,
-    pub uid: String,
-    pub namespace: String,
-    pub node: String,
-    pub time_used_tot: u64,
-    pub time_used_user_mode: u64,
-    pub time_used_system_mode: u64,
-    pub anon_used_mem: u64,
-    pub file_mem: u64,
-    pub shared_mem: u64,
-    pub file_mapped_mem: u64,
-    pub total_mem: u64,
+    pub name: String,               // Name of a Kubernetes pod
```

Comment:
```suggestion
    /// Name of a Kubernetes pod
    pub name: String,               
```
Here is how each field should be documented. In Rust, `//` is a regular comment (not in the doc), and `///` is a doc comment`. Every doc comment must be written just above the element that it documents.